x-pack/plugins/observability/public/components/observability_app_page_template.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import type { LazyObservabilityPageTemplateProps } from '@kbn/observability-shared-plugin/public';
+import { usePluginContext } from '../hooks/use_plugin_context';
+
+export function ObservabilityAppPageTemplate({
+  children,
+  ...rest
+}: LazyObservabilityPageTemplateProps & {
+  children: ReactNode;
+}) {
+  const { ObservabilityPageTemplate } = usePluginContext();
```

Comment:
Thought this was necessary at first to keep the flyout open on page transitions. It isn't really, but decided to keep it as it's still a nice clean up: all the other routes no longer need to have `const { ObservabilityPageTemplate } = usePluginContext();` and the import of `usePluginContext`.